eva_cttv_pipeline/evidence_string_generation/clinvar_to_evidence_strings.py

```diff
@@ -417,12 +388,10 @@ def load_efo_mapping(efo_mapping_file):
                     trait_2_efo[clinvar_name].append((ontology_id, ontology_label))
                 n_efo_mappings += 1
             else:
-                unavailable_efo.add(clinvar_name)
-
-    print(str(n_efo_mappings) + ' EFO mappings loaded')
-    print(str(len(unavailable_efo)) + ' urls without an actual valid EFO mapping')
-
-    return trait_2_efo, unavailable_efo
+                import pdb; pdb.set_trace()
```

Comment:
That's right. I replaced it with throwing a `ValueError`, because at this point in the pipeline all traits **must** have mappings, and they can only be absent if the input file is malformed.